### ISSUE TITLE
`--duration` More precise terminal message when tests with small duration are hidden

### ISSUE THREAD

**Issue by smarie** [pytest-dev/pytest#6940]
Currently when tests are skipped because of the `--duration` flag, a generic message is printed to terminal:

```
(0.00 durations hidden.  Use -vv to show these durations.)
```

I suggest to improve it slightly with the number of hidden tests, as well as precise threshold:

```
(7 items with duration < 0.005s hidden.  Use -vv to show these durations.)
```

What do you think ?

**Comment by smarie** [comment]
I included this suggestion in my current PR about improving duration precision. Let me know if I should take it back or change it.

**Comment by laurentbristiel** [comment]
I like the message improvement. Don't know how it could bother anyone, but who knows..

**Comment by Zac-HD** [comment]
I like it a lot!  Maybe tag that PR as `closes #6940` so that happens automatically?

**Comment by smarie** [comment]
I did already in the commit. https://github.com/pytest-dev/pytest/pull/6939/commits/8c041d3e0adaef892af84a09a6f3171a46336f8d

Thanks @laurentbristiel  and @Zac-HD  for the prompt feedback!

**Comment by smarie** [comment]
Fixed in #6939
